age/src/x25519.rs

```diff
@@ -146,7 +146,7 @@ impl crate::Identity for Identity {
 ///
 /// This recipient type is anonymous, in the sense that an attacker can't tell from the
 /// age-encrypted file alone if it is encrypted to a certain recipient.
-#[derive(Clone, PartialEq, Eq)]
+#[derive(Clone, PartialEq, Eq, Hash)]
```

Comment:
Please update `age/CHANGELOG.md` with this addition (change line 14 to add `, Hash` to the existing list I added for your previous PR).

Reply:
Fixed!